bugbug/model.py

```diff
@@ -254,8 +255,18 @@ def train(self, importance_cutoff=0.15):
 
         print(f"No confidence threshold - {len(y_test)} classified")
         confusion_matrix = metrics.confusion_matrix(y_test, y_pred, labels=class_names)
-        print(confusion_matrix)
         tracking_metrics["confusion_matrix"] = confusion_matrix.tolist()
+        for i in range(len(tracking_metrics["confusion_matrix"])):
+            tracking_metrics["confusion_matrix"][i].insert(0, class_names[i])
+        headers_for_visualisation = class_names.copy()
+        headers_for_visualisation.insert(0, "True/False")
+        print(
+            tabulate(
+                tracking_metrics["confusion_matrix"],
+                headers=headers_for_visualisation,
```

Comment:
You don't need to create a new variable for this, you can simply do:
```Python
headers=class_names + ["True/False"]
```

Reply:
This won't append in beginning but in the end. If that's okay ?

Reply:
You can do ["True/False"] + class_names to append in the beginning